### PR TITLE
Listing initiatives

### PR DESCRIPTION
**Pull request by KatiRG**
Test passes but breaks Making a suggestion test for no discernible reason.

### CODE REVIEW

**Comment by oliverbarnes** [comment]
What's the error? 

btw, see my edits to pull request title and description. more readable this way. good rule of thumb is to think of a new contributor coming in who might read this communication

**Comment by KatiRG** [comment]
not ok 2 PhantomJS 1.9 - Making a suggestion to an initiative Successfully
    ---
        message: >
            AssertionError: expected '' to equal 'Make sure there\'s a doctor available for house calls'
    ...
not ok 3 PhantomJS 1.9 - Making a suggestion to an initiative "after each" hook
    ---
        message: >
            Assertion Failed: calling set on destroyed object
        stack: >
            Error: Assertion Failed: calling set on destroyed object
                at http://localhost:7357/assets/vendor.js:13269
                at set (http://localhost:7357/assets/vendor.js:19612)
                at http://localhost:7357/assets/vendor.js:65383
                at http://localhost:7357/assets/vendor.js:67226
                at tryable (http://localhost:7357/assets/vendor.js:16404)
                at http://localhost:7357/assets/vendor.js:20923
                at suspendListeners (http://localhost:7357/assets/vendor.js:16413)
                at _suspendBeforeObservers (http://localhost:7357/assets/vendor.js:18715)
                at http://localhost:7357/assets/vendor.js:67227
                at tryable (http://localhost:7357/assets/vendor.js:16404)
                at http://localhost:7357/assets/vendor.js:20923
                at suspendListeners (http://localhost:7357/assets/vendor.js:16413)
                at _suspendObservers (http://localhost:7357/assets/vendor.js:18720)
                at http://localhost:7357/assets/vendor.js:67228
                at http://localhost:7357/assets/vendor.js:65384
                at http://localhost:7357/assets/vendor.js:69254
                at applyStr (http://localhost:7357/assets/vendor.js:21157)
                at sendEvent (http://localhost:7357/assets/vendor.js:16470)
                at notifyBeforeObservers (http://localhost:7357/assets/vendor.js:19368)
                at propertyWillChange (http://localhost:7357/assets/vendor.js:19198)
                at http://localhost:7357/assets/vendor.js:14864
                at set (http://localhost:7357/assets/vendor.js:19615)
                at http://localhost:7357/assets/vendor.js:66963
                at http://localhost:7357/assets/vendor.js:69701
                at http://localhost:7357/assets/vendor.js:17687
                at http://localhost:7357/assets/vendor.js:17530
                at http://localhost:7357/assets/vendor.js:17688
                at http://localhost:7357/assets/vendor.js:69702
                at http://localhost:7357/assets/vendor.js:69734
                at http://localhost:7357/assets/vendor.js:66970
                at http://localhost:7357/assets/vendor.js:68000
                at http://localhost:7357/assets/vendor.js:66794
                at http://localhost:7357/assets/vendor.js:66953
                at http://localhost:7357/assets/vendor.js:70721
    ...

ok about editing the title. i was not sure what the convention was...

**Comment by oliverbarnes** [comment]
similar to #34. if you uncomment that test (Supporting an initiative), does it pass?

**Comment by KatiRG** [comment]
? Supporting an initiative test is not commented ?

**Comment by oliverbarnes** [comment]
hmm, it is on [master](https://github.com/oliverbarnes/participate-frontend/blob/master/tests/acceptance/supporting-an-initiative-test.js)

File: tests/acceptance/listing-initiatives-test.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+    App = startApp();
+  },
+  teardown: function() {
+    Ember.run(App, 'destroy');
+  }
+});
+
+test('Successfully', function(){
+  visit('/initiatives').then(function() {
+    expect(find('.info').text()).to.equal('Public health clinic: Allocate compensation money to create a local public health clinic');
```
Comment (oliverbarnes) [inline tests/acceptance/listing-initiatives-test.js]: when testing listings, it's good to have 2 or 3 items in the list

**Comment by KatiRG** [comment]
making a suggestion test is commented out, not supporting an initiative. It's the latter that is breaking with 'listing initiatives test', not the former.

File: tests/acceptance/listing-initiatives-test.js (hunk 1 above)
Comment (KatiRG) [inline tests/acceptance/listing-initiatives-test.js]: do you have any other initiatives or should i make a few up?

**Comment by oliverbarnes** [comment]
not on master, here's the link again: https://github.com/oliverbarnes/participate-frontend/blob/master/tests/acceptance/supporting-an-initiative-test.js

if you pulled from master and had already uncommented it, and it's passing, it's all good. Just comment out the test that's breaking now, and update the issue #34 about it. Then update the pull request by amending the current commit (`git commit --amend`), also changing the commit message to a simple 'Listing initiatives', and pushing again. Then wait for the Travis build to give the green light :)

**Comment by KatiRG** [comment]
> if you pulled from master and had already uncommented it, and it's passing, it's all good. 

making suggestion test has to be commented out. So does supporting an initiative test.

> Just comment out the test that's breaking now, and update the issue #34 about it. 

done

> Then update the pull request by amending the current commit (git commit --amend), also changing the commit message to a simple 'Listing initiatives', and pushing again. Then wait for the Travis build to give the green light :)

it passed!

File: tests/acceptance/listing-initiatives-test.js (hunk 1 above)
Comment (oliverbarnes) [inline tests/acceptance/listing-initiatives-test.js]: make a few up! good to have variety here, helps us think the system

**Comment by KatiRG** [comment]
@oliverbarnes you can ignore the changelog PR from KatiRG patch branch. I've redone it the normal way from RGSofC-Participate which I think is clearer.

**Comment by oliverbarnes** [comment]
Almost there. Need to rebase against origin/master (git rebase -i origin/master) and squash the changelog commit against the main Listing initiatives commit.

**Comment by KatiRG** [comment]
Done as you requested. Please let me know if it's ready for merge :)

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,6 +1,8 @@
 08/20/2014
 ==========
 * [#38](https://github.com/oliverbarnes/participate-frontend/pull/38): First stab at styling with Bourbon and Bitters (Closes issue [#8](https://github.com/oliverbarnes/participate-frontend/issues/8))
+* [#36](https://github.com/oliverbarnes/participate-frontend/pull/36): Lists initiatives (Closes issue [#26](https://github.com/oliverbarnes/participate-frontend/issues/26))
+* [#31](https://github.com/oliverbarnes/participate-frontend/pull/31): Test validation for initiative title and description (Closes issue [#12](https://github.com/oliverbarnes/participate-frontend/issues/12))
```
Comment (oliverbarnes) [inline CHANGELOG.md]: This update (31) is from 08/12

Comment (KatiRG) [inline CHANGELOG.md]: fixed

**Comment by KatiRG** [comment]
Done, ready to be checked. Thanks!

**Comment by oliverbarnes** [comment]
:+1:

**Comment by KatiRG** [comment]
:)

**Pull request merged**